### PR TITLE
Add Setup page for cameras

### PR DESCRIPTION
**Pull request by Techno11**
camsetup

Also adds a new addon for pinging the cameras, switch, and mixer.

### CODE REVIEW

File: src/renderer/pages/camera_setup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                    Camera Setup
+                </Typography.Title>
+
+                <Flex gap="small" align="center" vertical>
+                    <Flex gap="small" wrap="wrap">
+                        <Button onClick={previous} disabled={subStep === 0}>
+                            Previous
+                        </Button>
+                        {subStep < subSteps.length - 1 && (
+                            <Button onClick={next} disabled={subStep === subSteps.length - 1}>
```
Comment (evanlihou) [inline src/renderer/pages/camera_setup/index.tsx:68]: Seems like disabled would never be true here?

```suggestion
                            <Button onClick={next}>
```

File: src/renderer/pages/camera_setup/index.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                <Typography.Text>
+                    <ReadyHandler
+                        ready={cam1 ? ReadyState.Ready : ReadyState.NotReady}
+                    />{' '}
+                    Camera 1
+                </Typography.Text>
+
+                <Typography.Text>
+                    <ReadyHandler
+                        ready={cam1 ? ReadyState.Ready : ReadyState.NotReady}
```
Comment (evanlihou) [inline src/renderer/pages/camera_setup/index.tsx:98]:
```suggestion
                        ready={cam2 ? ReadyState.Ready : ReadyState.NotReady}
```

**Review by evanlihou — COMMENTED** [review]
Good luck with those build errors... They don't look fun lol

**Comment by Techno11** [comment]
Build errors were 2-fold, tsc compiling things it didn't need to, and outdated React type libraries

**Review by evanlihou — APPROVED** [review]

**Pull request merged**